.github/workflows/windows-msvc.yml

```diff
@@ -41,8 +41,8 @@ jobs:
         target:
           - Debug
           - Release
-
     runs-on: ${{ matrix.os }}
+    timeout-minutes: 30
```

Comment:
Might be a bit short when dependencies are not cached (I remember runs between 20-30 min)

Reply:
All cases I've observed did not seem to exceed 20 to 25 minutes. I'll expand to 45 minutes for a better margin.